components/Snackbar/src/MDCSnackbarMessageView.h

```diff
@@ -54,4 +54,43 @@
  */
 @property(nonatomic, strong, nullable) UIFont *buttonFont UI_APPEARANCE_SELECTOR;
 
+/**
+ The color used for button text on the snackbar in normal state.
+
+ Default is white.
+ */
+@property(nonatomic, strong, nullable) UIColor *buttonTextColor UI_APPEARANCE_SELECTOR;
```

Comment:
buttonTitleForState:

If you strongly prefer two separate methods, please rename to match UIButton,
buttonTitleColor
highlightedTitleColor